### ISSUE TITLE
potential human error with respect to C-H bond for triple bonded carbon

### ISSUE THREAD

**Issue by davidlmobley** [openforcefield/smirnoff99Frosst#81]
Currently we have these bonds for C-H:
```
[#6X4:1]-[#1:2]                 340.0  1.090   parm99 aliphatic CH
[#6X3:1]-[#1:2]                 367.0  1.080   parm99 vinylic CH
[#6X2:1]-[#1:2]                 340.0  1.090   parm99 CZ-HZ alkyne #C-H
```

The comments in the FF and in @cbayly13 's conversion notes note that the triple bonded case (the last one) comes from the parm99 alkyne #C-H but parm99 has: `CZ-HZ  400.0    1.056`

Somehow this turned into:
`[#6X2:1]-[#1:2]                 340.0  1.090   parm99 CZ-HZ alkyne #C-H`

 I suspect this actually may be a copy and paste error, because the single bond is:
`[#6X4:1]-[#1:2]                 340.0  1.090   parm99 aliphatic CH`

so the parameters are exactly the same.

I think we should be fixing the relevant line by switching it to 
`[#6X2:1]-[#1:2]                 400.0    1.056   parm99 CZ-HZ alkyne #C-H`

@cbayly13 input?

**Comment by cbayly13** [comment]
@davidlmobley I agree I probably made a cut and paste error, and that the values we should put in smirnoff99Frosst should be
`[#6X2:1]-[#1:2] 400.0 1.056 parm99 CZ-HZ alkyne #C-H`

**Comment by bannanc** [comment]
I can do this update tomorrow and adjust the README (see issue #82)

**Comment by davidlmobley** [comment]
Great, thanks!

**Comment by bannanc** [comment]
This was fixed in PR #83 and is in the current release so I'm closing this issue.
